test/tests/functional/pbs_verify_log_output.py

```diff
@@ -152,3 +152,47 @@ def test_auto_sched_cycle_trigger(self):
                                          starttime=started_time)
             except PtlLogMatchError:
                 self.assertFalse(True)
+
+    def test_supported_auth_method_msgs(self):
+        """
+        Test case to verify pbs_supported_auth_method messages in the log
+        during start/restart or HUP the SERVER/COMM daemon
```

Comment:
Could be reworded to - Test to verify PBS_SUPPORTED_AUTH_METHODS is logged in server and comm daemon logs after start, restart and HUP.